website_base_multi_image/views/layout.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+<odoo>
+
+<template id="base_multi_image_fields">
+    <t t-set="owner_model" t-value="owner_model or 'product.template'"/>
```

Comment:
This is base module, so add no defaults please. Let it die if variable is missing.